blockbin/src/registerServiceWorker.js

```diff
@@ -0,0 +1,108 @@
+// In production, we register a service worker to serve assets from local cache.
```

Comment:
Do we actually care about supporting an offline mode?

You can't really read or write from the blockchain if you're offline right?

Reply:
This came with the boilerplate I used. I still thought it could be a nice add (for instance, if someone frequently visits something posted on blockbin, it will be cached and viewable offline).

Reply:
Fair enough let's leave that in ;)